curvefs/src/client/rpcclient/task_excutor.cpp

```diff
@@ -72,6 +79,41 @@ int TaskExecutor::DoRPCTask(std::shared_ptr<TaskContext> task) {
     return retCode;
 }
 
+void TaskExecutor::DoAsyncRPCTask(TaskExecutorDone *done) {
+    brpc::ClosureGuard done_guard(done);
+    task_->rpcTimeoutMs = opt_.rpcTimeoutMS;
+    int retCode = -1;
+
+    if (task_->retryTimes++ > opt_.maxRetry) {
+        LOG(ERROR) << task_->TaskContextStr()
+                   << " retry times exceeds the limit";
+        done->SetRetCode(retCode);
+        return;
+    }
+
+    if (!HasValidTarget() && !GetTarget()) {
+        LOG(WARNING) << "get target fail for " << task_->TaskContextStr()
+                     << ", sleep and retry";
+        bthread_usleep(opt_.retryIntervalUS);
```

Comment:
what's the purpose of sleep in here? I think is conflicting with sleep in line 210

Reply:
done